neuralmonkey/encoders/cnn_encoder.py

```diff
@@ -141,7 +139,8 @@ def __init__(self,
                 self.encoded = multilayer_projection(
                     last_layer_flat, fully_connected,
                     activation=tf.nn.relu,
-                    dropout_plc=self.dropout_placeholder)
+                    dropout_keep_prob=dropout_keep_prob,
```

Comment:
tady by asi mělo být spíš `self.dropout_keep_prob`, když se to tam na začátku přiřazuje. Teoreticky kdyby se volala nějaká funkce, která mění `self.dropout_keep_prob`, tak by se to tady neprojevovalo.